backend/sublet/views.py

```diff
@@ -104,14 +139,14 @@ def list(self, request, *args, **kwargs):
         baths = params.get("baths", None)
 
         queryset = Sublet.objects.all().filter(expires_at__gte=timezone.now())
-
         # Apply filters based on query parameters
         if title:
             queryset = queryset.filter(title__icontains=title)
         if address:
             queryset = queryset.filter(address__icontains=address)
         if amenities:
-            queryset = queryset.filter(amenities__name__in=amenities)
```

Comment:
I believe the deleted line is the functionality we want. If we provide a `amenities=["X", "Y"]` then the for loop you implemented would filter for `amenities__name="X" and amenities__name="Y"`, which will lead to the empty set.

Reply:
Uhhhh not sure, I got this from @vcai122 thoughts?

Reply:
Gut feeling is that you're right Justin, but maybe there's a reason Vincent asked me to make this change.

Reply:
justin said its fine